### ISSUE TITLE
inconsistent spellings 

### ISSUE THREAD

**Issue by jsaintlauren** [srophe/srophe-app-data#435]
@davidamichelson @nathangibson 
How do we want to handle it when headwords are spelled differently for saints' texts about the same person? 

e.g.

work 230 and work 1979 -- 
It is just an issue insofar as users browsing through the names..

**Comment by nathangibson** [comment]
@wsalesky Just discussed this -- would you please write a script that 
1. Goes through the BHSE works and looks in the names of the commemorated persons to find one that is contained in the English headword title of the work. 
2. Replaces that part of the title with the English headword of the commemorated person.

For example, for the work http://syriaca.org/work/1911 check the names in http://syriaca.org/person/1105 for any that are contained in "Abhaï (text)", then replace so that it is "Abhai (text)".

**Comment by wsalesky** [comment]
I'm a little confused. Am I replacing the text in the work record and the person record? Or just the person record? Are the current titles the gedesh version of the name? Or am I pattern matching on the word before the '(text)' ?

Thanks for the clarification!

**Comment by nathangibson** [comment]
We want to replace any occurrence of the person's name *in the syriaca-headword title of the work*. No changes should be made to the person record. 

In other words, in most BHSE works, the name of one or more commemorated persons appears as *part of* the headword title of the work. This name should already be in the person record, but it may not be the headword. So we want to cycle through the names of each commemorated person, checking if they are contained in the work headword title. If so, we want to replace just that portion of the title with the headword name text from the person record. Does that make sense?

**Comment by wsalesky** [comment]
I think so, but lets put it on the agenda for tomorrow, just to make sure I am understanding it correctly.

**Comment by jsaintlauren** [comment]
Will do!  I think that it will be easier to explain per conversation.

**Comment by wlpotter** [comment]
@jsaintlauren I don't think this go resolved?  http://syriaca.org/work/1911 and http://syriaca.org/person/1105 still have different forms of Abhai. Do we need to revisit?
